dev-tools/migrate-config.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# Script to assist with migrating kibana.yml to opensearch_dashboards.yml
```

Comment:
Can we add more details for this script? As a customer (e.g. an engineer that consumes Dashboards? Who else would use this?), when do I need to use this script? What can go wrong? How should I confirm the results?

Maybe the answers to these questions are trivial, but I wanted to pose them as someone who doesn't know what all of the use cases are.